lib/gitspindle/github.py

```diff
@@ -825,7 +823,8 @@ def issues(self, opts):
                 continue
             print(wrap("Issues for %s/%s" % (repo.owner.login, repo.name), attr.bright))
             for issue in issues:
-                url = issue.pull_request and issue.pull_request['html_url'] or issue.html_url
+                pr = issue.pull_request()
+                url = pr and pr.html_url or issue.html_url
```

Comment:
While we're at it, can we switch to the more modern `pr.html_url if pr else issue.html_url` syntax?

Reply:
done